R/settings_quarto_website.R

```diff
@@ -5,19 +5,12 @@
 
     # drop empty lines
     settings <- settings[!grepl("^\\w*$", settings)]
-
-    fn <- fs::path_join(c(path, "_quarto", "_quarto.yml"))
-    yaml::write_yaml(settings, fn, indent.mapping.sequence = TRUE)
-
-    # yaml::write_yaml converts true to yes, but quarto complains
-    settings <- .readlines(fn)
-    settings <- gsub(": yes$", ": true", settings)
-    settings <- gsub(": no$", ": false", settings)
-    writeLines(settings, fn)
-
-    tmp <- fs::path_join(c(path, "_quarto", "docs"))
-    fs::dir_copy(tmp, fs::path_join(c(path, "_quarto")), overwrite = TRUE)
-    fs::dir_delete(tmp)
+    settings = yaml::as.yaml(
+      settings, indent.mapping.sequence = TRUE, 
+      handler = list(logical = yaml::verbatim_logical)
+    )
+    settings = strsplit(settings, "\\n")[[1]]
```

Comment:
Small nitpick but can you use `<-` for assignment for consistency with the rest of the code? (I just make a comment here but there are other changes where you should make this replacement)